### PR TITLE
documentation(EJ2-65258): Review corrections addressed.

### PR DESCRIPTION
**Pull request by Deepika5698**
### Feature description

Need to include the review corrections for message feature list documentation.

### Analysis and design

- Add plural word (check the bold word) - In the below sample, the Message component content is customized with HTML elements and React Button components, which are directly added to the HTML element.
- To customize the Message component's appearance, add the custom class to the message through `cssClass` property. This custom class will be added to the root element. Based on this custom class, the user can override the styles of the message at the application level.
- Use the content in severity heading- The following example demonstrates the severities of the messages.
- Move the showIcon details to Icons md file under No Icon heading.
- Use the content- The variants of the message can be changed based on the variant property.
- Use the content-The following example demonstrates how the default message is rendered with a custom severity icon.
- Remove the custom appearance changes in custom icon heading sample. We will include it in customization heading.
- Maintain the following order in Icons md file.

   No Icon

   Custom Icon

   Close Icon

- Use the content- The Essential JS 2 Message has the following predefined CSS classes that can be defined in the HTML elements, which renders the message without any script reference.
- Remove the loading text from CSS Message heading. It has is displayed even displayed message content.

 
### Solution description
 
Added the reviewed corrections.
 
### Output screenshots
NA
 
### Areas affected and ensured
No area is affected due to this changes.
 
### Test cases
NA
 
### Test bed sample location
NA
 
### Additional checklist
 
* Did you run the automation against your implementation? NA
* Is there any API name change? No
* Is there any existing behavior change of other features due to this code change? No
* Does your new code introduce new warnings or binding errors? No
* Does your code pass all FxCop and StyleCop rules? Yes
* Did you record this case in the unit test or UI test? NA

### CODE REVIEW

File: ej2-asp-core-toc.html [hunk 1]
```diff
@@ -1394,13 +1394,27 @@
...
 		<ul>
-		<li><a href="/ej2-asp-core/message/getting-started">Getting Started</a></li>
-		<li><a href="/ej2-asp-core/message/severities">Severities</a></li>
-		<li><a href="/ej2-asp-core/message/variants">Variants</a></li>
-		<li><a href="/ej2-asp-core/message/icons">Icons</a></li>
-		<li><a href="/ej2-asp-core/message/customization">Customization</a></li>
-		<li><a href="/ej2-asp-core/message/template">Template</a></li>
-		<li><a href="/ej2-asp-core/message/accessibility">Accessibility</a></li>
+		<li>
+			<a href="/ej2-asp-core/message/getting-started">Getting Started</a>
```
Comment (Deepika5698) [inline ej2-asp-core-toc.html:1398]: Included this changes to resolve the below build error.
https://jenkins-ci.syncfusion.com/job/Content/job/ej2-asp-core-mvc-docs/1325/artifact/cireports/FilePathValidation/FilePathValidation.html

**Review by PiramanayagamSyncfusion — APPROVED** [review]
Changes fine

**Comment by Deepika5698** [comment]
rebuild

**Comment by Deepika5698** [comment]
I have moved this changes in below PR. So I am closing this PR.
PR link: https://github.com/syncfusion-content/ej2-asp-core-mvc-docs/pull/1085

**Pull request closed without merging**